README.md

```diff
@@ -26,6 +26,57 @@ Generates bindings file `path/to/uniffi/definitions/definitions.go`
 To integrate the bindings into your projects, simply add the generated bindings file to your project.
 Generated bindings require Go 1.19 or later to compile.
 
+
+# Configuration options
+
+It's possible to configure some settings by passing `--config` argument to the generator. All
+configuration keys are defined in `bindings.go` section.
+```bash
+uniffi-bindgen-go path/to/definitions.udl --config path/to/uniffi.toml
+```
+
+- `package_name` - override the go package name.
+
+- `cdylib_name` - override the dynamic library name linked by generated bindings, excluding `lib`
+    prefix and `.dll` file extension.
+    ```toml
+    # For library `libgreeter.dll`
+    [bindings.go]
+    cdylib_name = "greeter"
+    ```
+
```

Comment:
`cdylib_name` does not exist for Go.
```suggestion
```

Reply:
Pretty sure it is used now in the current impl

Reply:
ah, understood, removed it